src/App.php

```diff
@@ -94,7 +94,9 @@ public function getContainer()
     private function getDispatcher()
     {
         $container = $this->container;
-
+        if (!is_callable($container->get("dispatcher"))) {
+            throw new \RuntimeException("Dispatcher must be a callable");
+        }
```

Comment:
:+1: for this. Also because single quotes doesn't charge the lexer with extra job.